seed/lib/superperms/orgs/models.py

```diff
@@ -140,14 +139,16 @@ def save(self, *args, **kwargs):
         # Create a default cycle for the organization if there isn't one already
         from seed.models import Cycle
         year = date.today().year - 1
-        cycle_name = 'Default ' + str(year) + ' Calendar Year'
+        cycle_name = '{} Calendar Year'.format(year)
         if not Cycle.objects.filter(name=cycle_name, organization=self).exists():
```

Comment:
I know that I updated this in the bricr-models branch. Shouldn't conflict too much though. Any reason to remove the "Default"? You just want it to be cleaner?